app/views/layouts/_sidebar.html.erb

```diff
@@ -257,6 +257,7 @@
       <div class="Docs__nav__section-content">
         <ul class="Docs__nav__sub-nav">
           <%= sidebar_link_to "Managing API Tokens", 'apis/managing-api-tokens' %>
+          <%= sidebar_link_to "API Disparity", 'apis/api-differences' %>
```

Comment:
This doesn't seem to work? ?

Reply:
Oops. Moved the file to a proper place. Thanks!